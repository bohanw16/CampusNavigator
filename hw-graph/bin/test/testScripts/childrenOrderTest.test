# childrenOrder testing file
# Tests that a graph can be created,
# two nodes can be added to the graph,
# two edges between them can be added,
# and the children of the parent node can be printed out in correct order

# Create a graph
CreateGraph g1

# Add two nodes
AddNode g1 n1
AddNode g1 n2

# Add two edges
AddEdge g1 n1 n2 e1
AddEdge g1 n1 n2 e2

# Print the nodes in the graph and the outgoing edges from n1
ListNodes g1
ListChildren g1 n1