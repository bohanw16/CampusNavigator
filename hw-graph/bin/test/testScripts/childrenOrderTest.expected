# childrenOrder testing file
# Tests that a graph can be created,
# two nodes can be added to the graph,
# two edges between them can be added,
# and the children of the parent node can be printed out in correct order

# Create a graph
created graph g1

# Add two nodes
added node n1 to g1
added node n2 to g1

# Add two edges
added edge e1 from n1 to n2 in g1
added edge e2 from n1 to n2 in g1

# Print the nodes in the graph and the outgoing edges from n1
g1 contains: n1 n2
the children of n1 in g1 are: n2(e1) n2(e2)